demo-project/src/demo_project/pipelines/feature_engineering/nodes.py

```diff
@@ -99,3 +100,19 @@ def joiner(spine_df: pd.DataFrame, *dfs: pd.DataFrame) -> pd.DataFrame:
     # Confirm that the number of rows is unchanged after the operation has completed
     assert spine_df.shape[0] == merged_dfs.shape[0]
     return merged_dfs
+
+
+def create_feature_importance(data: pd.DataFrame):
+    feature_name = []
+    feature_score = []
+    for i in range(0, 15):
+        feature_name.append("feature_" + str(i))
+        feature_score.append(random.randint(0, 999) / 1000)
+    feature_importance_df = pd.DataFrame(
+        {"Features": feature_name, "Score": feature_score}
+    )
+    random_number = random.randint(0, 100) / 200 + random.randint(0, 100) / 200
+    feature_importance_df["Score"] = feature_importance_df["Score"].apply(
```

Comment:
I don't understand why there's so many different `random.randint`s in this function to generate a single random number? 🤔 Why do we need anything more than just the original `feature_score` random number on L110?

Reply:
you are right. initially I was reading from a csv hence I had the randomness and then i just created an in memory array. so i don't need to repeat it. i have removed this